app/services/submit_reference.rb

```diff
@@ -7,33 +7,8 @@ def initialize(reference:)
   end
 
   def save!
-    if enough_references_have_been_submitted?
-      # With the decoupled_references feature active, we should be preventing
-      # submission of the application unless the MINIMUM_COMPLETE_REFERENCES
-      # have been provided. We also no longer allow the candidate to manually
-      # mark the section complete - it is considered complete when the minimum
-      # references are given. We check the references directly when attempting
-      # to submit the form, rather than inspecting the
-      # ApplicationForm#references_completed flag.
-      #
-      # As such, this method only needs to update the state of the reference,
-      # not the application form.
-      #
-      # TODO: drop the ApplicationForm#references_completed column when
```

Comment:
Do we have this carded up somewhere?

Reply:
Not that I'm aware of, but I'm working on it now. as part of the overarching card.

Reply:
It's done here https://github.com/DFE-Digital/apply-for-teacher-training/pull/3234